ims/ostis_tech/semantic_networks_processing/section_basic_programming_language_scp/section_basic_programming_language_scp_content.scs

```diff
@@ -3431,7 +3431,7 @@ rrel_access
  		*);
 	<= nrel_subdividing: 
```

Comment:
this shouldn't be a subdividing, it is possible that arc will be a membership arc, but without positivity specified

Reply:
ok, turns out this file has several 'access sc-arc' mentions